Release/1.0.0

mzeong opened the pull request.

mzeong committed:
[FEAT] 소셜 로그인 API 구현 (#2)

* Feat: 카카오 로그인 API 구현 #1

* Feat: access token 재발급 API 구현 #1

* Feat: access token 검증 미들웨어 구현 #1

* Feat: 구글 로그인 API 구현 #1

* Feat: 네이버 로그인 API 구현 #1

mzeong committed:
Feat: team, member 테이블 설계

mzeong committed:
[FEAT] 종목별 나의 팀 조회 API 구현 (#5)

* Feat: 라우트 추가 #4

* Feat: 컨트롤러 함수 추가 #4

* Feat: Team 테이블에 category 필드 추가 #4

* Feat: 서비스 함수 추가 #4

* Feat: DAO 함수 추가 #4

mzeong committed:
[FEAT] 팀 상세 조회 API 구현 (#7)

* Feat: 라우트 함수 추가 #6

* Feat: 컨트롤러 함수 추가 #6

* Feat: 서비스 함수 추가 #6

* Feat: DAO 함수 추가 #6

* Feat: DTO 함수 추가 #6

mzeong committed:
[FEAT] 기존 팀 추가 API 구현 (#9)

* Feat: 라우트 추가 #8

* Feat: 라우트 함수 추가 #8

* Feat: 컨트롤러 함수 추가 #8

* Feat: 서비스 함수, 에러 코드 추가 #8

* Feat: DAO 함수 추가 #8

mzeong committed:
[FEAT] 새로운 팀 생성 API 구현 (#12)

* Feat: 라우트 추가 #8

* Feat: 라우트 함수 추가 #8

* Feat: 컨트롤러 함수 추가 #8

* Feat: 서비스 함수, 에러 코드 추가 #8

* Feat: DAO 함수 추가 #8

* Feat: 라우트 함수 추가 #3

* Feat: 컨트롤러 함수 추가 #3

* Feat: request body 유효성 검사 #3

* Feat: 서비스 함수 추가 #3

* Feat: DAO 함수 추가 #3

mzeong committed:
[FEAT] 날짜별 게스트 모집글 조회 API 구현 (#14)

* [FEAT]게스트모집 조회  API 구현

* Feat: guest, guestUser 테이블 생성 #11

* Feat: 라우트 연결 #11

* Feat: guest, guest user 테이블 생성 #11

* Feat: 날짜별 게스트 모집글 조회 #11

* Style: guest user 테이블 이름 언더스코어 처리 #11

* Feat: guest 테이블 모집 인원 수 필드 추가 #11

* Feat: DAO 함수 추가 #11

* Feat: 팀의 멤버 수 조회하는 함수 추가 #11

* Feat: 서비스 함수 추가 #11

* Feat: DTO 함수 추가 #11

* Feat: API와 상관 없는 코드 주석 처리 #11

---------

Co-authored-by: ByeonYujin124 <byj7176@gmail.com>

mzeong committed:
[FEAT] 팀 수정 API 구현 (#16)

* Feat: 라우트 함수 추가 #13

* Feat: 컨트롤러 함수 추가 #13

* Feat: 서비스 함수 추가 #13

* Feat: error status 추가 #13

* Feat: UpdateTeamSchema 추가 #13

* Feat: DAO 함수 추가 #13

mzeong committed:
Feat: 성별, 연령대, 레벨을 숫자에서 문자열로 변환하도록 readGuestingResponseDTO 수정 #11

mzeong committed:
[FEAT] 커뮤니티 글 목록 조회 API 구현 (#18)

* Feat: community post, image, comment, bookmark 테이블 생성 #17

* Style: dto 파일 이름 변경 #17

* Feat: verifyUserIfExists 미들웨어 추가 #17

* Feat: 라우트 추가 #17

* Style: dto 파일 이름 변경에 따른 경로 변경 #17

* Feat: 컨트롤러 함수 추가 #17

* Feat: 서비스 함수 추가 #17

* Feat: DAO 함수 추가 #17

* Feat: DTO 함수 추가 #17

mzeong committed:
[FEAT] 커뮤니티 글 저장과 저장 취소 API 구현 (#20)

* Feat: 라우트 함수 추가 #19

* Feat: 컨트롤러 함수 추가 #19

* Feat: 서비스 함수 추가 #19

* Feat: DAO 함수 추가 #19

mzeong committed:
[FEAT] 커뮤니티 글 본문 조회 API 구현 (#22)

* Feat: 라우트 함수 추가 #21

* Feat: 컨트롤러 함수 추가 #21

* Feat: 서비스 함수, 에러 코드 추가 #21

* Feat: DAO 함수 추가 #21

* Feat: DTO 함수 추가 #21

mzeong committed:
Fix: readCommunityPostResponseDTO에 comment id 추가 #21

mzeong committed:
[FEAT] 커뮤니티 댓글 추가 조회 API 구현 (#24)

* Feat: 라우트 함수 추가 #23

* Feat: 컨트롤러 함수 추가 #23

* Feat: 서비스 함수 추가 #23

* Fix: 조회한 댓글 수가 아니라 전체 댓글 수를 응답하도록 수정 #21

mzeong committed:
[FEAT] 커뮤니티 글 작성 API 구현 (#26)

* Feat: 라우트 함수 추가 #25

* Feat: 컨트롤러 함수 추가 #25

* Feat: CreateCommunityPostSchema 생성 #25

* Feat: 서비스 함수 추가 #25

* Feat: DAO 함수 추가 #25

mzeong committed:
[FEAT] 커뮤니티 댓글 작성 API 구현 (#28)

* Feat: 라우트 함수 추가 #27

* Feat: 컨트롤러 함수 추가 #27

* Feat: CreateCommunityCommentSchema 생성 #27

* Feat: 서비스 함수 추가 #27

* Feat: DAO 함수 추가 #27

mzeong committed:
[FEAT] 내가 쓴 글 조회 API 구현 (#31)

* Feat: 커뮤니티와 대관정보 통합한 post, image, bookmark 테이블 생성 #29

* Feat: 라우트 추가 #29

* Feat: 컨트롤러 함수 추가 #29

* Feat: 서비스 함수 추가 #29

* Feat: DAO 함수 추가 #29

* Feat: DTO 함수 추가 #29

mzeong committed:
Refactor: 커뮤니티 글 목록 조회 API 구현 #17

mzeong committed:
[FEAT] 저장한 글 조회 API 구현 (#33)

* Feat: 라우트 함수 추가 #32

* Feat: 컨트롤러 함수 추가 #32

* Feat: 서비스 함수 추가 #32

* Feat: DAO 함수 추가 #32

mzeong committed:
Refactor: 커뮤니티 글 저장과 저장 취소 API 구현 #19

ujiiin committed:
FEAT) 게스트 모집글 지역 필터링 API 구현 #15

ujiiin committed:
FEAT) 게스트 모집글 성별 필터링 API 구현 #35

ujiiin committed:
FEAT) 게스트 모집글 레벨 필터링 API 구현 #36

mzeong committed:
Refactor: 카카오 로그인 API 구현 #1

mzeong committed:
Feat: 내가 쓴 글 조회 API 엔드포인트의 URI 변경 #29

mzeong committed:
Refactor: 커뮤니티 글 본문 조회 API 구현 #21

mzeong committed:
Refactor: 커뮤니티 글 작성 API 구현 #25

mzeong committed:
Refactor: 커뮤니티 댓글 작성 API 구현 #27

mzeong committed:
Refactor: 커뮤니티 댓글 추가 조회 API 구현 #23

mzeong committed:
[FEAT] 날짜별 연습경기 모집글 조회, 연습경기 모집글 필터링(지역, 레벨, 성별) API 구현 (#43)

* Feat: game 테이블 설계 #10

* Feat: 라우트 추가 #10

* Feat: 라우트 함수 추가 #10

* Feat: 컨트롤러 함수 추가 #10

* Fix: Team, Game 테이블 수정 #10

* Fix: 컨트롤러 함수 수정 #10

* Feat: 서비스 함수 추가 #10

* Feat: DAO 함수 추가 #10

* Fix: 날짜별 조회 함수 수정 #10

* Fix: 서버 에러 수정 #10

* Feat: 연습경기 조회 최신순 정렬 추가 #10

* Fix: Game, Team 모델 수정 #10

* Feat: 모집 상태(status) 출력 #10

* Feat: 연습경기 모집 상태(status) 구현 #10

---------

Co-authored-by: Doeun <doeun990802@gmail.com>

ujiiin committed:
[FEAT] 게스트 모집 조회, 신청 API 구현

mzeong committed:
Refactor: 네이버 로그인 API 구현 #1

mzeong committed:
[FEAT] 로그아웃 API 구현 (#45)

* Feat: 라우트 함수 추가 #44

* Feat: 컨트롤러 함수 추가 #44

* Feat: 서비스 함수 추가 #44

* Feat: setRefreshToken 함수의 refreshToken 매개변수 타입 변경 #44

doeunyy committed:
[Feat] 연습경기 신청 가능 팀 목록 조회 #46 (#47)

* Feat: games 라우트 추가 #46

* Feat: games 컨트롤러 함수 추가 #46

* Feat: teams 서비스 함수 추가 #46

* Feat: teams DAO 함수 추가 #46

* Chore: package-lock.json 업데이트 #46

doeunyy committed:
[FEAT] 연습경기 모집글 상세 조회 API 구현 (#50)

* Feat: 라우트 추가 #48

* Feat: 컨트롤러 함수 추가 #48

* Feat: 서비스 함수 추가 #48

* Feat: DAO 함수 추가 #48

* Feat: DTO 함수 추가 #48

* Fix: 팀원 목록 리더 중복 제거 #48

doeunyy committed:
[FEAT] 연습경기 모집글 작성 API 구현 (#52)

* Feat: 라우트 추가 #51

* Feat: 컨트롤러 함수 추가 #51

* Feat: 스키마 작성 #51

* Feat: game 모델 수정 #51

* Feat: 서비스 함수 추가 #51

* Feat: DAO 함수 추가 #51

* Feat: team DAO - getTeamCategoryByLeaderId 함수 추가 #51

doeunyy committed:
[FEAT] 연습경기 모집글 수정 API 구현 (#54)

* Feat: 라우트 추가 #53

* Feat: 컨트롤러 함수 추가 #53

* Feat: 스키마 추가 #53

* Feat: game response error 추가 #53

* Feat: 서비스 함수 추가 #53

* Feat: DAO 함수 추가 #53

mzeong committed:
[FEAT] 종목별 선수 정보 수정 API 구현 (#55)

* Feat: 라우트 추가 #49

* Feat: body, query, params 검증하는 validate 함수 생성 #49

* Feat: profile 테이블 생성, user 테이블 수정 #49

* Refactor: 성별, 연령대, 종목 constant 수정 및 enum 파일 분리 #49

* Feat: update user profile schema 생성 #49

* Feat: team 테이블의 gender, ageGroup 필드 타입 변경 #49

* Style: enum 멤버의 이름을 첫 글자만 대문자로 변경 #49

* Feat: 컨트롤러 함수 추가 #49

* Style: enum 멤버의 이름 변경에 따른 코드 수정 #49

* Style: constant 함수의 이름 변경에 따른 코드 수정 #49

* Feat: 서비스 함수와 DAO 함수 추가 #49

* Style: constant 함수의 이름 변경에 따른 코드 수정 #49

mzeong committed:
[FEAT] 종목별 선수 정보 조회 API 구현 (#42)

* Feat: profile 테이블 생성 #34

* Feat: 라우트 추가 #34

* Feat: validateParams 함수 생성 #34

* Feat: 컨트롤러 함수 추가 #34

* Feat: 서비스 함수 추가 #34

* Feat: DAO 함수 추가 #34

* Feat: DTO 함수 추가 #34

mzeong committed:
Fix: 종목별 선수 정보 조회 API 구현 머지 과정에서 중복 코드 제거 #34

mzeong committed:
Feat: users 라우트, 컨트롤러, 서비스, DAO 타입 지정 및 이름 수정 #34 #49

mzeong committed:
[FEAT] 게스트 신청 API 구현 (#57)

* FEAT) 게스트 모집글 상세조회 API 구현 #37

* FEAT) 게스트 모집글 작성 API 구현 #38

* FEAT) 게스트 모집글 작성 API 수정 #38

* FEAT) 게스트 모집글 수정 API 구현 #39

* FEAT) 게스트 신청 API 구현 #40

* Fix: where parameter "id" has invalid "undefined" value 에러 해결 #37

* Fix: Incorrect integer value: '[object Promise]' for column 'team_id' 에러 해결 #38

* fix: GUEST_NOT_FOUND 발생 해결 #39

* fix) 게스트 모집글 작성 API TEAM 유무 확인 추가 #38

* Fix) 게스트 신청 API 승인여부 컬럼 추가

* Chore: 사용하지 않는 라이브러리 제거

* Fix: 사용할 수 없는 함수 제거

---------

Co-authored-by: ByeonYujin124 <byj7176@gmail.com>
Co-authored-by: ByeonYujin124 <52813483+ByeonYujin124@users.noreply.github.com>

mzeong committed:
Feat: 날짜별 게스팅 내역 조회 API 구현 커밋 복원 #41

ujiiin committed:
Merge pull request #59 from TiimMate/feature/41

[FEAT] 날짜별 게스팅 내역 조회 API 구현

mzeong committed:
Feat: 날짜별 호스팅 내역 조회 API 구현 커밋 복원 #58 (#60)

mzeong committed:
Refactor: teams 라우트, 컨트롤러, 서비스, DAO, DTO 타입 지정 및 이름 수정 #3 #4 #6 #13

mzeong committed:
Refactor: validate 함수에 사용할 수 있도록 post, comment schema 수정 #25 #27

mzeong committed:
Feat: createMemberSchema 생성 #8

doeunyy committed:
Refactor: validate 함수 사용 가능하도록 game schema 수정 #51 #53 (#64)

doeunyy committed:
[FEAT] 대관정보 글 작성 API 구현 (#67)

* Feat: 라우트 추가 #66

* Feat: 컨트롤러 함수 추가 #66

doeunyy committed:
Revert "[FEAT] 대관정보 글 작성 API 구현" (#69)

doeunyy committed:
[FEAT] 대관정보 글 작성 API 구현 (#70)

* Feat: 라우트 추가 #66

* Feat: 컨트롤러 함수 추가 #66

* Fix: 컨트롤러 함수 수정 #66

* Feat: 서비스 함수 추가 #66

doeunyy committed:
[FEAT] 대관정보 글 목록 조회 API 구현 (#71)

* Feat: 라우트 추가 #68

* Feat: 컨트롤러 함수 추가 #68

* Feat: 서비스 함수 추가 #68

mzeong committed:
Refactor: 구글 로그인 API 구현 #1

doeunyy committed:
[FEAT] 연습경기 신청 API 구현 (#74)

* Feat: 라우트 추가 #65

* Feat: 컨트롤러 함수 추가 #65

* Feat: game-apply 스키마 & 필드 추가 #65

* Feat: 서비스 함수 추가 #65

* Feat: DAO 함수 추가 #65

* Feat: DAO 함수 수정 #65

doeunyy committed:
[FEAT] 호스팅 신청팀 목록 조회 (#76)

* Feat: 라우트 추가 #68

* Feat: 컨트롤러 함수 추가 #68

* Feat: 서비스 함수 추가 #68

* Feat: 라우트 추가 #68

* Feat: 컨트롤러 함수 추가 #68

* Feat: 서비스 함수 추가 #68

* Feat: DAO 함수 추가 #68

ujiiin committed:
매칭내역 API 구현 (#78)

* FEAT) 게스트 모집글 상세조회 API 구현 #37

* FEAT) 게스트 모집글 작성 API 구현 #38

* FEAT) 게스트 모집글 작성 API 수정 #38

* FEAT) 게스트 모집글 수정 API 구현 #39

* FEAT) 게스트 신청 API 구현 #40

* Fix: where parameter "id" has invalid "undefined" value 에러 해결 #37

* Fix: Incorrect integer value: '[object Promise]' for column 'team_id' 에러 해결 #38

* fix: GUEST_NOT_FOUND 발생 해결 #39

* fix) 게스트 모집글 작성 API TEAM 유무 확인 추가 #38

* Fix) 게스트 신청 API 승인여부 컬럼 추가

* [FEAT] 날짜별 호스팅 내역 조회 API 구현 #58

* Fix) 날짜별 호스팅 내역 조회 API 연습게임 조회 추가 #58

* [FEAT] 호스팅 신청자 목록 조회 #72

* [FEAT] 게스트 신청자 승인 #75

* Fix) 날짜별 게스팅 내역 조회 API 연습게임 조회 추가 #41

* Fix) varifyUser 추가 #41

---------

Co-authored-by: 김민정 <mjeong@sookmyung.ac.kr>

doeunyy committed:
[FEAT] 신청팀 승인 API 구현 (#80)

* Feat: 라우트 추가 #77

* Feat: 컨트롤러 함수 추가 #77

* Feat: 서비스 함수 추가 #77

* Feat: DAO 함수 추가 #77

mzeong committed:
[FEAT] 팀 리뷰 작성 API 구현 (#81)

* Feat: team_review 테이블 생성 #61

* Feat: schema 생성 #61

* Feat: 라우트 추가 #61

* Feat: 컨트롤러 함수 추가 #61

* Feat: schema에 team_match_id 필드 추가 #61

* Style: schema 필드명 수정 #61

* Feat: games DAO 함수 작성, getGameById 함수 이름 getGameByUserId으로 변경 #61

* Feat: DAO 함수 추가 #61

* Feat: 서비스 함수 추가, 응답 에러 추가 #61

doeunyy committed:
Feat: 날아간 코드 복구 #73 (#82)

mzeong committed:
[FEAT] 게스트 리뷰 작성 API 구현 (#83)

* Feat: schema 추가 #62

* Feat: 라우트 함수 추가 #62

* Feat: 컨트롤러 함수 추가 #62

* Fix: schema 필드명 guest_match_id에서 guestMatchId로 수정 #62

* Feat: schema에 revieweeId 필드 추가 #62

* Feat: user_review 테이블 생성 #62

* Style: model 파일 이름 일관성 있게 수정 #62

* Feat: DAO 함수 추가 #62

* Feat: 서비스 함수 추가 #62

* Fix: model 파일의 model name, table name 수정 #62

* Fix: getLeaderId 함수가 leaderId를 반환하도록 수정 #62

ujiiin committed:
Refactor: guest, matching API 중 변경된 함수 및 타입 수정 (#85)

mzeong committed:
Merge branch 'develop' into release/1.0.0

mzeong reviewed (commented):
@ByeonYujin124 게스트 모집글 작성 API

mzeong reviewed (commented):
@doeunyy 연습경기 모집글 상세 조회, 날짜별 연습경기 모집글 조회, 연습경기 모집글 성별 필터링, 연습경기 모집글 레벨 필터링, 연습경기 모집글 지역 필터링 API

doeunyy reviewed (commented):
@ByeonYujin124 게스트 신청(날짜별 게스트 모집글 조회, 게스트 모집글 레벨/성별/지역 필터링, 게스트 모집글 상세 조회, 게스트 신청)

mzeong reviewed (commented):
@ByeonYujin124 날짜별 호스팅 내역 조회, 날짜별 게스팅 내역 조회 API

doeunyy reviewed (commented):
@mzeong 리뷰 작성 (게스트 리뷰 작성, 팀 리뷰 작성)

mzeong reviewed (commented):
@doeunyy 호스팅 신청팀 목록 조회 API

doeunyy reviewed (commented):
@mzeong 마이페이지 (종목별 선수 정보 조회 및 수정)

mzeong reviewed (commented):
@ByeonYujin124 호스팅 신청자 목록 조회 API

mzeong reviewed (commented):
@doeunyy 신청팀 승인 API

mzeong reviewed (commented):
@ByeonYujin124 신청자 승인 API

mzeong reviewed (commented):
@ByeonYujin124 게스트 모집글 수정 API

mzeong committed:
Fix: 카카오 로그인 KOE010 에러 해결 #91

mzeong committed:
Merge pull request #92 from TiimMate/bugfix/91

[BUGFIX] 카카오 로그인 KOE010 에러

Commit:
Feat: game-DTO 관련 코드 수정

ujiiin committed:
Fix) guest schema 추가 및 validate 설정 변경 #97

ujiiin committed:
Fix) guest schema 추가 및 validate 설정 변경

mzeong committed:
Fix: 종목별 나의 팀 조회 API Schema 변경 #96

ujiiin committed:
Merge branch 'bugfix/validate-guests-route' of https://github.com/TiimMate/TeamMateServer into bugfix/validate-guests-route

doeunyy committed:
Merge pull request #98 from TiimMate/bugfix/96

[BUGFIX] 종목별 나의 팀 조회 API Schema 변경

Commit:
Refactor: game 모델 applyTeamId 필드 삭제 #102

mzeong committed:
Merge pull request #104 from TiimMate/refactor/102

[REFACTOR] game 모델 apply_team_id 필드 삭제

ujiiin committed:
[BUGFIX] 게스트 모집글 작성 및 신청 API 수정 #103

mzeong committed:
Refactor: Record를 이용한 Statuses로 변경

mzeong committed:
Style: games.dao.ts 파일명 game.dao.ts로 변경

mzeong committed:
Refactor: 의미 없는 주석과 console.log 제거

mzeong committed:
Fix: gameId 유효성 검증

mzeong committed:
Feat: readGameDetailResponseDTO에 getGender, getAgeGroup 함수 적용

mzeong committed:
Merge pull request #94 from TiimMate/bugfix/modify-games-dto

[BUGFIX] game-DTO 관련 코드 수정

ujiiin committed:
Merge branch 'release/1.0.0' into bugfix/validate-guests-route

mzeong committed:
Fix: 게스트 모집글 작성 service 함수 수정 #107

ujiiin committed:
[BUGFIX] 게스트 모집글 신청 에러 케이스 추가 #107

ujiiin committed:
[BUGFIX] guest, guest-user dao파일 구분 #110

ujiiin committed:
[BUGFIX] 매칭내역 조회 타입 추가 #110

ujiiin committed:
[BUGFIX] guestUser status constant 추가  #110

ujiiin committed:
[BUGFIX] 멤버 카운트 함수 추가 #110

ujiiin committed:
[BUGFIX] 멤버 함수 매칭내역 API에 적용 #110

mzeong committed:
Fix: fields.ts 파일 내 날아간 코드 복구 #107

mzeong committed:
Fix: 게스트 신청 전 선수 정보 미입력인지 검증 #107

mzeong committed:
Fix: request query가 아닌 DB 조회를 통한 category 사용 #107

mzeong committed:
Merge pull request #107 from TiimMate/bugfix/validate-guests-route

Bugfix/게스트 모집글 작성 및 신청 API 수정

mzeong committed:
Fix: matchings.dao.ts 파일 42번째 줄 오타 수정 #112

mzeong committed:
Merge pull request #113 from TiimMate/bugfix/112

[BUGFIX] matchings.dao.ts 파일 내에 오타 수정

mzeong committed:
Refactor: findTeamPreviewByCategory 함수에서 id도 함께 조회하도록 수정 #109

mzeong committed:
Feat: 리뷰 작성 가능한 기한을 시합 시간 이후부터 한 달 동안으로 제한 #88

doeunyy committed:
Merge pull request #114 from TiimMate/refactor/109

[REFACTOR] 종목별 나의 팀 조회 API 응답 수정

Commit:
Bugfix: gameApply 모델 추가 및 ORM 적용 #105

mzeong committed:
Merge branch 'release/1.0.0' into bugfix/105

mzeong committed:
Merge pull request #117 from TiimMate/bugfix/105

[BUGFIX] gameApply 모델 추가 및 ORM 적용

Commit:
Merge remote-tracking branch 'origin/release/1.0.0' into bugfix/118

Commit:
Bugfix: 신청팀 승인 API 메서드 변경 #118

doeunyy committed:
Merge pull request #115 from TiimMate/feature/88

[FEAT] 리뷰 작성 가능한 기한 설정

Commit:
Bugfix: 연습경기 신청 팀 리더 검증 및 ORM 적용 #121

Commit:
Merge branch 'bugfix/121' into bugfix/123

Commit:
Bugfix: games.route.ts 파일에 validate 함수 추가 #123

ujiiin committed:
[BUGFIX] main branch pull 중 에러 해결 #103

ujiiin committed:
[BUGFIX] 게스팅 목록 조회 및 신청 에러 해결 #103

ujiiin committed:
[BUGFIX] main branch pull 중 충돌 해결 #110

ujiiin committed:
[BUGFIX] 매칭내역 조회 에러 해결 #110

ujiiin committed:
[REFACTOR] 게스트 목록 조회 API 페이징 추가 #125

ujiiin committed:
[REFACTOR] 게스트 목록 조회 API 빠진 파일 추가 #125

mzeong committed:
Fix: 게스트 모집글 작성, 수정, 상세 조회, 신청은 로그인된 유저만 가능하도록 수정 #110

mzeong committed:
Fix: 매칭내역 페이지 내 모든 API는 로그인된 유저만 가능하도록 수정 #110

mzeong committed:
Fix: guests.controller.ts 파일 수정 #110

mzeong committed:
Fix: getTeamDetailForGuesting 함수 수정 #110

mzeong committed:
Fix: findMemberInfoByCategory 함수를 통한 멤버 프로필 조회 #110

mzeong committed:
Style: insertGuestUser로 함수 이름 변경 #110

mzeong committed:
Style: 더 이상 사용하지 않는 findMemberInfoByTeamId 함수 제거 #110

mzeong committed:
Comment: addMemberCount 함수에 경고 메세지 추가 #110

mzeong committed:
Merge pull request #119 from TiimMate/bugfix/118

[BUGFIX] 신청팀 승인 API 메서드 변경

ujiiin committed:
Fix: 유저(팀장)의 모든 카테고리 게스팅, 호스팅 내역 조회 가능하도록 수정 #110

ujiiin committed:
Fix: 게스트 모집글 수정 API에서 teamId 가져오는 부분 수정 #110

mzeong committed:
Fix: 날짜별 호스팅 내역 조회 API 수정 #110

- 유저가 팀장인 팀이 게스트를 모집한 경기, 상대팀을 모집한 경기를 조회하는 API
- 유저가 팀장인 팀을 모두 조회하도록 수정
- findGamesOfMatchingHosting 함수에서 game, team 테이블 간 조회가 안 되는 문제 해결
- findGuestsOfMatchingHosting, findGamesOfMatchingHosting 함수 이름 변경 및 파일 이동
- readMatchingHostingResponseDTO 내에서 경기시간 순 정렬
- 경기 타입을 이넘으로 관리

mzeong committed:
Merge branch 'bugfix/110' of https://github.com/TiimMate/TeamMateServer into bugfix/110

mzeong committed:
Fix: 날짜별 게스팅 내역 조회 API 수정 #110

- 유저가 게스트로 신청한 경기, 유저가 팀장인 팀이 상대팀으로 신청한 경기를 조회하는 API

mzeong committed:
Merge pull request #111 from TiimMate/bugfix/110

[BUGFIX] 매칭내역 조회 API 수정

ujiiin committed:
Merge branch 'release/1.0.0' of https://github.com/TiimMate/TeamMateServer into refactor/125 #125

ujiiin committed:
Fix: level type string->int 수정 #125

ujiiin committed:
Refactor: getGuestingById 함수 변경 #128

ujiiin committed:
[FEAT] 게스트 테이블에 status 필드 추가 #128

ujiiin committed:
[FEAT] 게스트 신청 유저수에 따른 Guest.status 값 판별 #128

ujiiin committed:
Refactor: updateGuestUserStatus 함수 matchings.service -> guests.service 이동 #128

ujiiin committed:
Refactor: Guest.status dto에 적용 #128

ujiiin committed:
Fix: Guest.status 값 판별 오류 수정 #128

doeunyy committed:
Merge pull request #130 from TiimMate/feature/128

[FEAT] 게스트 마감 필드 추가

doeunyy committed:
Merge branch 'release/1.0.0' into refactor/125

doeunyy committed:
Merge pull request #129 from TiimMate/refactor/125

[REFACTOR] 게스트 목록 조회 API 페이징 추가

ujiiin committed:
Refactor: 게스트 모집글 조회 레벨 필터링 API  level 범위 수정 #132

ujiiin committed:
Refactor: 연습게임 조회 레벨 필터링 API level 범위 수정 #132

ujiiin committed:
Refactor: getLevelById 함수 사용하는 부분 삭제 #132

ujiiin committed:
Refactor: defaultLevel 사용하는 부분 삭제 #132

ujiiin committed:
Refactor: level.constant.ts 삭제 #132

Commit:
Feat: 대관정보에 대관일자/대관장소/대관모집상태 추가 #131

ujiiin committed:
[FEAT] Guest 모델 gameDuration 필드 추가 #133

ujiiin committed:
[FEAT] guest 조회, 작성 API에 gameDuration 추가 #133

ujiiin committed:
[FEAT] 매칭내역 조회 API에 gameDuration 추가 #133

doeunyy committed:
Merge pull request #134 from TiimMate/refactor/132

[REFACTOR] 레벨 범위 변경으로 인한 필터링 수정

doeunyy committed:
Merge pull request #137 from TiimMate/feature/133

[FEAT] 게스트 API 게임 소요시간 필드 추가

mzeong committed:
Feat: user 테이블에 avatarUrl 필드 추가 #116

mzeong committed:
Feat: avatarUrl 필드 타입 수정 #116

mzeong committed:
Feat: schema에 avatarUrlField 추가 #116

mzeong committed:
Feat: 종목별 선수 정보 조회 API 응답에 avatarUrl 추가 #116

doeunyy committed:
Merge pull request #138 from TiimMate/feature/116

[FEAT] 종목별 선수 정보 수정 및 조회 API에 프로필 사진 포함

ujiiin committed:
Fix: post.dao 불필요한 부분 삭제 #131

ujiiin committed:
Merge branch 'release/1.0.0' into feature/131

ujiiin committed:
Merge pull request #135 from TiimMate/feature/131

[FEAT] 대관정보에 대관일자/대관장소/대관모집상태 추가 #131

Commit:
Merge remote-tracking branch 'origin/release/1.0.0' into feature/136

Commit:
Feat: 대관정보 글 목록 조회 시 글 내용/모집상태 추가 #136

Commit:
Merge remote-tracking branch 'origin/release/1.0.0' into bugfix/121

mzeong committed:
Merge pull request #124 from TiimMate/bugfix/123

[BUGFIX] games.route.ts 파일에 validate 함수 추가

Commit:
Bugfix: 연습경기 신청 에러 수정 #144

Commit:
Bugfix: 연습경기 신청 에러 수정 #144

Commit:
Bugfix: 연습경기 신청 가능 팀 조회 엔드포인트 변경 및 API 응답 수정 #146

mzeong committed:
Refactor: 서비스 함수 수정 #141

mzeong committed:
Refactor: 응답에 유저 아이디, 프로필 사진 포함 #141

mzeong committed:
Refactor: getTeamDetail 함수에서 Member 테이블 조인 제거 #141

mzeong committed:
Refactor: DTO 수정 #141

mzeong committed:
Feat: 더 이상 사용하지 않는 readMembersInfo 함수 제거 #141

mzeong committed:
Feat: getTeamDetail 함수에서 초대 코드, 성별, 연령대, 지역, 체육관명 추가로 조회 #141

mzeong committed:
Refactor: DTO 함수에 MemberInfo 인터페이스 추가 #141

mzeong committed:
Refactor: DTO 내에 console.log 제거 #141

mzeong committed:
Feat: DAO, DTO에 id, height 추가 #140

mzeong committed:
Merge pull request #147 from TiimMate/bugfix/146

[BUGFIX] 연습경기 신청 가능 팀 조회 엔드포인트 변경 및 API 응답 수정

mzeong committed:
Refactor: userId path variable 추가 #150

doeunyy committed:
Merge pull request #148 from TiimMate/feature/141

[REFACTOR] 팀 상세 조회 API 응답 수정

doeunyy committed:
Merge pull request #149 from TiimMate/refactor/140

[REFACTOR] 종목별 선수 정보 조회 API 응답 수정

doeunyy committed:
Merge pull request #151 from TiimMate/refactor/150

[REFACTOR] 종목별 선수 정보 조회 API 엔드포인트 수정

mzeong committed:
Merge pull request #145 from TiimMate/bugfix/144

[BUGFIX] 연습경기 신청 에러 수정

mzeong committed:
Fix: 엔드포인트 오타 수정 #152

trad-fh committed:
Merge pull request #153 from TiimMate/bugfix/152

[BUGFIX] 종목별 선수 정보 조회 API 엔드포인트 오타 수정

ujiiin committed:
[FEAT] 게스트 모집글 조회 API guest.id 추가 #155

ujiiin committed:
[FEAT] 게스트 상세 조회 API guest.description 추가 #155

Commit:
Feat: 연습경기 API 게임 소요시간 필드 추가 및 연습경기 목록 조회 id 추가 #154

Commit:
Refactor: 연습경기 모집글 상세조회 API 응답 수정 #160

Commit:
Merge branch 'feature/154' into refactor/162

Commit:
Merge branch 'refactor/160' into refactor/162

Commit:
Refactor: 연습경기 모집글 상세조회 수정 #162

ujiiin committed:
[FEAT] 게스트 상세조회, 호스팅 신청자 조회 avatarUrl 추가 #143

mzeong committed:
Merge pull request #139 from TiimMate/feature/136

[FEAT] 대관정보 글 목록 조회 시 글 내용/모집상태 추가

mzeong committed:
Merge pull request #156 from TiimMate/feat/155

[FEAT] 게스트 조회 데이터 추가

mzeong committed:
Merge pull request #159 from TiimMate/feature/154

[FEAT] 연습경기 API 게임 소요시간 필드 추가 및 연습경기 목록 조회 id 추가

mzeong committed:
Merge pull request #161 from TiimMate/refactor/160

[REFACTOR] 연습경기 모집글 상세조회 API 응답 수정

mzeong committed:
Merge branch 'release/1.0.0' into refactor/162

mzeong committed:
Merge pull request #163 from TiimMate/refactor/162

[REFACTOR] 연습경기 모집글 상세조회 수정

mzeong committed:
Merge pull request #164 from TiimMate/feat/143

[FEAT] 게스트 상세 조회, 호스팅 신청자 조회 응답에 프로필 추가
